addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -409,34 +488,45 @@ <H2>VIEWSTATE</H2>
 The "Viewstate without MAC Signature (Unsure)" alert will only be raised at LOW threshold.
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ViewstateScanRule.java">ViewstateScanRule.java</a>
+<br>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10032/">10032</a>.
 
-<H2>X-AspNet-Version Response Header</H2>
+<H2 id = "id-10061">X-AspNet-Version Response Header</H2>
 This checks response headers for the presence of X-AspNet-Version/X-AspNetMvc-Version details.
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XAspNetVersionScanRule.java">XAspNetVersionScanRule.java</a>
+<br>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10061/">10061</a>.
 
-<H2>X-Backend-Server Header Information Leak</H2>
+<H2 id = "id-10039">X-Backend-Server Header Information Leak</H2>
 This checks response headers for the presence of X-Backend-Server details.
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XBackendServerInformationLeak.java">XBackendServerInformationLeak.java</a>
+<br>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10039/">10039</a>.
 
-<H2>X-ChromeLogger-Data Header Information Leak</H2>
+<H2 id = "id-10052">X-ChromeLogger-Data Header Information Leak</H2>
 This checks response headers for the presence of X-ChromeLogger-Data or X-ChromePhp-Data details.
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XChromeLoggerDataInfoLeakScanRule.java">XChromeLoggerDataInfoLeakScanRule.java</a>
+<br>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10052/">10052</a>.
 
-<H2>X-Content-Type-Options</H2>
+<H2 id = "id-10021">X-Content-Type-Options</H2>
 This scan rule check for the Anti-MIME-Sniffing header X-Content-Type-Options and ensures it is set to 'nosniff'.<br>
 At MEDIUM and HIGH thresholds this rule does not alert on client or server error responses or redirects.<br> 
 At LOW threshold it will alert on all responses including errors and redirects.<br>
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/XContentTypeOptionsScanRule.java">XContentTypeOptionsScanRule.java</a>
+<br>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10021/">10021</a>.
 
-<H2>X-Debug-Token Information Leak</H2>
+<H2 id = "id-10052">X-Debug-Token Information Leak</H2>
```

Comment:
I'd suggest removing the spaces around the equals signs

Reply:
sure sir